doc/rst/usingchapel/QUICKSTART.rst

```diff
@@ -160,28 +160,28 @@ What's next?
 =============================================== =========================
 **For more detailed information about:**        **refer to:**
 ----------------------------------------------- -------------------------
-    platform-specific notes                     `platforms`_
+    platform-specific notes                     `doc/rst/platforms`_
     online Chapel Documentation                 `chapel.cray.com/docs`_
     example Chapel programs                     ``examples/README``
-    setting Chapel environment variables        `doc/chplenv.rst`_
-    building the compiler                       `doc/building.rst`_
-    compiling Chapel programs                   `doc/compiling.rst`_
-    executing Chapel programs                   `doc/executing.rst`_
-    debugging Chapel programs                   `doc/debugging.rst`_
-    reporting bugs                              `doc/bugs.rst`_
+    setting Chapel environment variables        `doc/rst/usingchapel/chplenv.rst`_
```

Comment:
Yeah, I noted this in the commit message. I'd be OK with a mix of relative paths for things like `chplenv.rst` and `../platforms`, and absolute paths for top-level items: `$CHPL_HOME/PERFORMANCE.md`.